libs/dm-localization/src/lib/Data/sheet.ts

```diff
@@ -84,12 +89,21 @@ const data = {
     vaporize: 3111567595,
     melt: 3497793651,
     // Technically this should be 'rupture', but I don't see a 'rupture' string
-    bloom: 3593652061, // or maybe it is 4074539307
+    bloom: 1216168406,
     burgeon: 2021015747,
     hyperbloom: 2697794115,
     quicken: 347446227,
     aggravate: 2961120579,
-    spread: 2191579811, // or maybe it is 982642368
+    spread: 2191579811,
+  },
+  stat: {
```

Comment:
i only added the necessary things because i'm gonna work with this further in another PR. Will leave the bulk of changes until then.